homeassistant/components/steam_online/strings.json

```diff
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Use https://steamid.io to find your Steam account ID",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "account": "Steam account ID"
+        }
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "The Steam integration needs to be manually re-authenticated\n\nYou can find your key here: https://steamcommunity.com/dev/apikey"
```

Comment:
URLs and similar data should not be included directly in the translation string but shown via `description_placeholders`, to avoid translating the data.